### PR TITLE
Make it compile and show size

### PR DESCRIPTION
**Pull request by behdad**

### CODE REVIEW

**Comment by behdad** [comment]
Sample output of `make` now:
```
size emoji_presentation_scanner.o emoji_presentation_scanner_vs.o
   text	  data	   bss	   dec	   hex	filename
    917	     0	     0	   917	   395	emoji_presentation_scanner.o
   1114	     0	     0	  1114	   45a	emoji_presentation_scanner_vs.o
```

So, the `_vs` version is only 200bytes larger. I think we should merge them.

**Comment by behdad** [comment]
@tursunova Any chance you can fold these small changes onto your PR? My ragel is different from yours so I let you commit the final files. Thanks.

File: emoji_presentation_scanner_vs.rl [hunk 1]
```diff
@@ -109,5 +116,5 @@ scan_emoji_presentation_vs (emoji_text_iter_t p,
   /* Should not be reached. */
   *is_emoji = false;
   *has_vs = false;
-  return pe;
+  return p;
```
Comment (drott) [inline emoji_presentation_scanner_vs.rl:119]: Why is this line changed?

Comment (behdad) [reply emoji_presentation_scanner_vs.rl:119]: Because return value is non-const, and `pe` is const. The code says `/* Should not be reached */`. Maybe we should also add an assert there.

**Comment by tursunova** [comment]
Oops, I'm really sorry @behdad, I have missed this one. I can generate files with Ragel 6.10 and push the changes here if it's okay? Sorry.

**Comment by behdad** [comment]
Sounds good, if you don't mind resolving the conflicts.

**Comment by tursunova** [comment]
Sorry @behdad , I think I don't have an access to push commits in this PR, so I created new one https://github.com/google/emoji-segmenter/pull/9, hope it's okay? Or could you please give me an access to push in your repo, then I will push changes in this pr?

**Comment by behdad** [comment]
Thanks Munira. I can do the rest from here.

**Comment by tursunova** [comment]
Thanks Behdad, created a new pr for `EMOJI_LINKAGE` https://github.com/google/emoji-segmenter/pull/9#issuecomment-2337781674

**Review by kimfuller138 — APPROVED** [review]
